tests/listobjects/matrix_ttus.go

```diff
@@ -0,0 +1,56 @@
+package listobjects
+
+import (
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+
+	listobjectstest "github.com/openfga/openfga/internal/test/listobjects"
+)
+
+var ttus = []matrixTest{
+	{
+		Name: "ttus_alg_combined",
+		Tuples: []*openfgav1.TupleKey{
+			// Satisfies left side `and_ttu`
+			{Object: "directs:ttu_alg_1", Relation: "direct_mult_types", User: "user:ttu_anne"},
+			{Object: "directs:ttu_alg_1", Relation: "other_rel", User: "user:*"},
+
+			{Object: "ttus:ttu_alg_1", Relation: "mult_parent_types", User: "directs:ttu_alg_1"},
+			{Object: "ttus:ttu_alg_1", Relation: "direct_parent", User: "directs:ttu_alg_1"},
+
```

Comment:
should we also have case where it satisfy via ttu_alg_combined_computed path?  More specifically, where the parent on the left side of BUT NOT is subject to condition in the `mult_parent_types` relation